azure/durable_functions/models/TaskOrchestrationExecutor.py

```diff
@@ -68,6 +68,21 @@ def execute(self, context: DurableOrchestrationContext,
         self.context = context
         evaluated_user_code = fn(context)
 
+        # The minimum History size is 2, in the shape: [OrchestratorStarted, ExecutionStarted].
+        # At the start of replay, the `is_replaying` flag is determined from the
+        # ExecutionStarted event.
+        # For some reason, OrchestratorStarted does not update its `isPlayed` field.
+        if len(history) < 2:
+            err_message = "Internal Durable Functions error: "\
+                + f"received History array of size {len(history)} "\
+                + "when a minimum size of 2 is expected. "\
+                + "Please report this issue in our GitHub repo."
```

Comment:
Maybe include the URL so that users don't have to guess?
```suggestion
                + "Please report this issue at https://github.com/Azure/azure-functions-durable-python/issues."
```